ui/app/components/copyable-code/index.hbs

```diff
@@ -7,6 +7,6 @@
    class={{if this.copySuccess 'copy-success'}}
    title="Copy"
   >
-    <Pds::Icon @type={{if this.copySuccess 'copy-success' 'copy-action'}} class="icon" />
+    <FlightIcon @name={{if this.copySuccess 'copy-success' 'copy-action'}} class="icon" />
```

Comment:
Looks like the icon is missing here:

<img width="386" alt="CleanShot 2021-12-09 at 10 07 20@2x" src="https://user-images.githubusercontent.com/34030/145376312-6aaf7b00-dcd0-4fcd-a40a-372122a15132.png">